unitTesting/test_control_manager.py

```diff
@@ -0,0 +1,48 @@
+from dynamic_graph.sot.core.matrix_util import matrixToTuple, vectorToTuple,rotate, matrixToRPY
+from dynamic_graph.sot.torque_control.control_manager import *
+from numpy import matrix, identity, zeros, eye, array, pi, ndarray, ones
+
+# Instanciate the free flyer
+cm = ControlManager("cm_test")
+
+q=zeros(36)
+dq=zeros(30)
+bemfFactor=ones(30)
+max_current=30.0*ones(30)
+max_tau = 100.0*ones(30)
+percentageDriverDeadZoneCompensation = 20.0*ones(30)
+signWindowsFilterSize = ones(30)
+tau = 100.0 * ones(30)
+tau_predicted = 110.0 * ones(30)
+pwmDes = 100.0 *ones(30)
+currentDes = 100.0 *ones(30)
+
+# Setting the robot configuration
+cm.base6d_encoders.value=q
```

Comment:
Does this mean that in sot v3 you can directly set the value of a signal to a numpy array? Do tuples still work?